components/Result/Result.scss

```diff
@@ -0,0 +1,89 @@
+.result {
+  display: none;
+
+  &__wrapper {
+    @include tablet {
+      display: flex;
+      align-items: center;
+      gap: 24px;
+    }
+
+    @include desktop-xl {
+      gap: 32px;
+    }
+  }
+
+  &__cards {
+    @include tablet {
+      margin-left: auto;
+      width: calc(50% - 12px);
+
+      & + & {
+        margin-top: 24px;
+      }
+    }
+
+    @include desktop-xl {
+      width: calc(50% - 20px);
+
+      & + & {
+        margin-top: 32px;
+      }
+    }
+  }
+
+  &__image {
+    @include tablet {
+      position: relative;
+      width: 340px;
+      height: 284px;
+    }
+
+    @include tablet-xl {
+      height: 352px;
+      min-width: 436px;
+    }
+
+    @include desktop {
+      height: 530px;
+      min-width: 596px;
+    }
+
+    @include desktop-xl {
+      height: 646px;
+      min-width: 712px;
+    }
+  }
+
+  &__background {
+    @include tablet {
+      position: absolute;
+      right: 0;
```

Comment:
you need to put these two properties on the class, since they are used for all devices. 
We leave the rest of the properties in media queries, because they change depending on the device